master/docs/developer/secrets.rst

```diff
@@ -50,14 +50,13 @@ Vault provider
 
 .. code-block:: python
 
-    c['secretsProviders'] = [secrets.SecretInVault(vaultToken=open('VAULT_TOKEN').read(),
-                                                vaultServer="http://localhost:8200",
-                                                apiVersion=2)]
+    c['secretsProviders'] = [secrets.SecretInVaultKv(authenticator=secrets.VaultAuthenticatorApprole(roleId="xxx", secretId="yyy"),
```

Comment:
I think this should be HashiCorpVaultKvSecretProvider. The previous value is an error.

Reply:
You are right, I tested it and buildbot throws an error.